house/views.py

```diff
@@ -42,11 +42,34 @@ def house_view(request):
 
     house = user.house
     expenses_list = Expenses.objects.filter(house_name=house)
-    context = {'house': house,
-               'house_expenses': expenses_list}
+    context = {'house': house, 'house_expenses': expenses_list}
     return render(request, MINE_MINE_PAGE_ROUTE, context)
 
 
+def add_house(request):
```

Comment:
`@login_required
def add_house(request):`
please add the login required to validate that only login uses can access this url